tests/requirements.txt

```diff
@@ -4,4 +4,5 @@ nose-cov
 python-coveralls
 coverage
 SQLAlchemy
-pyexcel-xls
+pyexcel-xls==0.0.7
+-e git+https://github.com/chfw/pyexcel.git@040cd392ec40069ac1a0f6df755694c8216366d8#egg=pyexcel-master
```

Comment:
Shall we have this instead?

   https://github.com/chfw/pyexcel/archive/master.zip

Reference:
http://stackoverflow.com/questions/20101834/pip-install-from-github-repo-branch